modules/volunteers/admin/sitesupport/vue.js

```diff
@@ -61,6 +66,50 @@ var app = Vue.createApp({
           const data = JSON.parse(response.responseText);
           this.reward_groups = data.data;
         })
+    },
+    clearMember() {
+      this.userId = null;
+      this.userData = null;
+      this.$refs.vhtbl.load();
+      this.$refs.gfttbl.load();
+      this.$refs.lookup.clear();
+    },
+    enterAdmin() {
+      var target = '';
```

Comment:
Can we use `let` here instead of `var`?

Additionally this could be a good use case for `URLSearchParams`:

```
const searchParams = new URLSearchParams('Function=volunteers/admin');

if (this.userId) {
  searchParams.append(`volunteerId=${this.userId}`);
}

const target = `index.php?${searchParams.toString()}`;
```

Reply:
This is a bit funky.

I am getting redirects looking like this `http://localhost:8080/index.php?Function=volunteers%2Fadmin&volunteerId=2001`  They appear to work but is that `%2F` going to be an issue

Reply:
The `%2F` in this case is the URL encoding, it should represent `/`. Does it still direct to the correct location? What potential issues do you foresee with the encoded value?

Another option if we want to keep the `/`' in the URL is to skip the URLSearchParams but things up a bit.

```
let target = 'index.php?Function=volunteers/admin`

if (this.userId) {
  target += `&volunteerId=${this.userId}`;
}